ui/components/component-library/button-base/button-base.scss

```diff
@@ -5,6 +5,13 @@
   vertical-align: middle;
   user-select: none;
 
+  // Focus styles from monorepo button
+  &:focus-visible {
+    outline: 2px solid var(--color-primary-default);
+    outline-offset: 2px;
+    box-shadow: none;
+  }
+
```

Comment:
Adding focus state to component-library button cc @brianacnguyen 


https://github.com/user-attachments/assets/23c49459-4f84-41b1-8897-3c5885c0fd20